src/observe-tag.js

```diff
@@ -109,17 +112,25 @@ module.exports = tagFunction => {
 					}
 
 					// also set the scrollLeft and scrollTop (since this is reset to 0 by default)
-					elementToGiveFocus.scrollLeft = removedElementWithFocusData.scrollLeft
-					elementToGiveFocus.scrollTop = removedElementWithFocusData.scrollTop
+					if (removedElementWithFocusData.scrollLeft || removedElementWithFocusData.scrollTop) {
+						elementToGiveFocus.scrollLeft = removedElementWithFocusData.scrollLeft
+						elementToGiveFocus.scrollTop = removedElementWithFocusData.scrollTop
+					}
```

Comment:
we verify that there is a `scrollLeft` and `scrollTop` now, previously this was causing a bug on elements that could have focus, but can't be scrolled.

Reply:
Weird wording question. scrollLeft makes sense, but scrollTop seems a bit weird. scrollToTop sounds better?

Reply:
Those are built into the DOM, so we can't really pick and choose those :man_shrugging: